regions/shapes/annulus.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import math
+import operator
+
+import numpy as np
+
+from astropy import wcs
+from astropy import coordinates
+from astropy import units as u
+
+from ..core import CompoundPixelRegion, CompoundSkyRegion
+from ..shapes import CirclePixelRegion, CircleSkyRegion
+
+
+class AnnulusPixelRegion(CompoundPixelRegion):
+    """
+    An annulus in pixel coordinates.
+
+    Parameters
+    ----------
+    center : :class:`~regions.core.pixcoord.PixCoord`
+        The position of the center of the annulus.
+    inner_radius : float
+        The inner radius of the annulus
+    outer_radius : float
+        The outer radius of the annulus
+    """
+
+    def __init__(self, center, inner_radius, outer_radius, meta=None, visual=None):
```

Comment:
A ds9 annulus can have an arbitrary number of rings.  I think an in/out annulus, as you've implemented, belongs here, but we also need something to handle the n-annuli case